OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -169,7 +175,13 @@ public void SendAirstrike(Actor self, WPos target, bool randomize = true, int at
 					attack.OnExitedAttackRange += onExitRange;
 					attack.OnRemovedFromWorld += onRemovedFromWorld;
 
-					a.QueueActivity(new Fly(a, Target.FromPos(target + spawnOffset)));
+					for (var strikes = 0; strikes < info.Strikes; strikes++)
+					{
+						a.QueueActivity(new Fly(a, Target.FromPos(target + spawnOffset)));
+						if (info.Strikes > 1)
+							a.QueueActivity(new FlyIdle(a, info.CircleDelay));
```

Comment:
Using `FlyTimed` here while keeping everything else the same actually produces much better results for the passes and the angles (provided the delay is still 30 ticks) that both look better and produces attack angles much closer to the original. I didn't verify if the angles are exactly like the original or the delay needs a bit of tweaking, but is probably close enough.

Reply:
Updated with your suggestions.